src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html">
+    <meta charset="utf-8">
+    <meta name="veiwport" content="width=device-width, initial-scale=1.0">
+    <title>title</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="hello">Hello, world!</div>
+  </body>
+</html>
```

Comment:
Always add empty line at the end of each file.

About why it is necessary - you can read here:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline